run

```diff
@@ -693,6 +705,11 @@ build() {
   ret=$?
   check_exit_code $ret "Error building application."
 
+  if [ $benchmark == true ]; then
+    app_source_root_path=$(get_app_source_root_dir $1)
+    run_command benchmarks/holoscan_flow_benchmarking/restore_application.sh ${app_source_root_path}
```

Comment:
It's not clear to me whether restoring the application after building is the right action or not. The source won't be an exact reflection of the built app, and I could see it being confusing for debugging. But then, interactive debugging doesn't necessarily make sense during benchmarking anyway.

I think it's fine to leave in for now and revisit if we encounter any issues.